web-client/src/presenter/actions/updatePractitionerUserAction.ts

```diff
@@ -13,7 +14,10 @@ export const updatePractitionerUserAction = async ({
   get,
   path,
 }: ActionProps) => {
-  const user = get(state.form);
+  const formState = get(state.form);
+  const user = new Practitioner(formState, {
+    applicationContext,
+  }).toRawObject({ removeValidationProperties: false });
```

Comment:
For posterity: this is being used to strip out `openCaseInfo` and `closedCaseInfo` from being passed to the backend, solving the bug